plugins/modules/postgresql_db.py

```diff
@@ -71,8 +71,16 @@
     - Supported compression formats for dump and restore include C(.pgc), C(.bz2), C(.gz) and C(.xz).
     - Supported formats for dump and restore include C(.sql) and C(.tar).
     - "Restore program is selected by target file format: C(.tar) and C(.pgc) are handled by pg_restore, other with pgsql."
+    - "."
```

Comment:
Why is here a `"."`?
I do not understand that, did I missed something?

Reply:
it's to add a bit of space between the lines to logically divide them. I tried `-""`  or `-" "` and sanity tests fail. If you know a better way, please share

Reply:
Okay, that's fine for me. I do not know a better solution.